app/models/search_facet.rb

```diff
@@ -255,13 +283,40 @@ def is_numeric?
     self.data_type == 'number'
   end
 
+  # know if a facet needs unit conversion
+  def must_convert?
```

Comment:
Nice foresight in supporting unit-less numeric metadata, e.g. `bmi` and `number_of_reads`.